packages/models-library/src/models_library/utils/change_case.py

```diff
@@ -0,0 +1,34 @@
+""" String convesion
+
+
+Example of usage in pydantic:
+
+[...]
+    class Config:
+        extra = Extra.forbid
+        alias_generator = snake_to_camel  # <--------
+        json_loads = orjson.loads
+        json_dumps = json_dumps
+
+"""
+# Partially taken from  https://github.com/autoferrit/python-change-case/blob/master/change_case/change_case.py#L131
+import re
+
+_underscorer1 = re.compile(r"(.)([A-Z][a-z]+)")
+_underscorer2 = re.compile(r"([a-z0-9])([A-Z])")
+
+
+def snake_to_camel(subject: str) -> str:
+    """ """
+    parts = subject.lower().split("_")
+    return parts[0] + "".join(word.title() for word in parts[1:])
+
+
+def snake_to_upper_camel(subject: str) -> str:
+    parts = subject.lower().split("_")
+    return "".join(word.title() for word in parts)
+
+
+def camel_to_snake(subject: str) -> str:
+    subbed = _underscorer1.sub(r"\1_\2", subject)
+    return _underscorer2.sub(r"\1_\2", subbed).lower()
```

Comment:
ok I see you created a utils.
Did you replace all occurences in the code?
```packages/models-library/src/models_library/utils/converters.py``` I created one camel_to_snake lately. and searching in the code shows at least:
```packages/simcore-sdk/tests/unit/test_node_ports_v2_port.py```
```services/api-server/tools/gen_api.py```
```services/web/server/src/simcore_service_webserver/utils.py```
and the change_case library in the webserver.

Also my version of camel to snake does not use a regex, which I read is supposed to be more efficient... but I don't really care.

Reply:
Yes, this is one step towards unifying this repetitions since we had (with yours) already three different versions of it.

This is a code that existed since very long in ``services/web/server/src/simcore_service_webserver/utils.py`` and decided to move it up to ``servicelib`` so we can use it when needed. The idea is to replace them all by thisone and add there possible extensions